tools/benchmark/common.slang

```diff
@@ -0,0 +1,336 @@
+/******************************************************************************
+ * Copyright (c) 2019-2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
We cannot have a copyrighted code in the master whose license is MIT.
This file has to go to `external` directory as an independent repository.